janusgraph-core/src/main/java/org/janusgraph/graphdb/olap/job/IndexRemoveJob.java

```diff
@@ -53,7 +54,7 @@
  */
 public class IndexRemoveJob extends IndexUpdateJob implements ScanJob {
```

Comment:
nitpick: Should we use here also the AbstractScanJob?

Reply:
I cannot do that since IndexRemoveJob already extends IndexUpdateJob. I cannot make IndexUpdateJob extend AbstractScanJob either, because one of its subclasses, IndexRepairJob, implements VertexScanJob interface rather than ScanJob interface.